pkgs/applications/misc/joplin-desktop/default.nix

```
@@ -51,6 +51,9 @@ let
     extraPkgs = appimageTools.defaultFhsEnvArgs.multiPkgs;
     extraInstallCommands = ''
       mv $out/bin/{${name},${pname}}
+      source "${makeWrapper}/nix-support/setup-hook"
+      wrapProgram $out/bin/${pname} \
+        --add-flags "\''${NIXOS_OZONE_WL:+\''${WAYLAND_DISPLAY:+--ozone-platform=wayland}}"
```

> grepping for add-flags.+NIXOS_OZON this is more commonly used nowadays

https://github.com/NixOS/nixpkgs/pull/203828#discussion_r1161018340

```suggestion
        --add-flags "\''${NIXOS_OZONE_WL:+\''${WAYLAND_DISPLAY:+--ozone-platform-hint=auto --enable-features=WaylandWindowDecorations}}"
```

---

The flags depend on the Electron version used, I tested it with those flags but Joplin just crashes